src/index.ts

```diff
@@ -72,6 +72,19 @@ export class WebRTCPlayer extends EventEmitter {
   }
 
   private async onConnectionStateChange(e) {
+
+    if (this.peer.connectionState === 'connected') {
+      const senders = await this.peer.getSenders();
+      if (senders) {
+        senders.forEach(sender => {
+          if (sender.track.readyState === 'ended') {
+            this.log('Track ended', e);
```

Comment:
We should also dispatch an event so any developer using this code can detect this change.